Mjolnir/src/OI.h

```diff
@@ -35,11 +35,48 @@ class OI {
 public:
 	OI();
 
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=PROTOTYPES
-
-	std::shared_ptr<Joystick> getdriver();
-	std::shared_ptr<Joystick> getcoDriver();
+	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=PROTOTYP
 	// END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=PROTOTYPES
+	static std::shared_ptr<OI> self;
```

Comment:
Both OI.h and OI.cpp don't look right. I'll take a look and try to make some changes.

The goal should be:
* No `shared_ptr`
* No `static`
* No public instances variables (e.g. `dA`, `dB`, ..., `dRB`)